test/integration/check/test_app/correct_syntax.lua

```diff
@@ -0,0 +1,3 @@
+do
+    print(2+2)
```

Comment:
As far as I understood, `a = 2 + 2` is enough (don't forget about spaces).
I just want to avoid unnecessary output to stdout.